### PR TITLE
14.ansible.ws

### PR DESCRIPTION
**Pull request by iryna-kurhuzenkava**

### CODE REVIEW

**Comment by iryna-kurhuzenkava** [comment]
14.ansible.ws + 12.ansible.run

File: iryna_kurhuzenkava/14.ansible.ws/ans_ws/roles/redmine_db/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    login_unix_socket: /var/run/mysqld/mysqld.sock
+
+- mysql_user:
+    name: "{{ app_db_user }}"
+    password: "{{ app_db_pass }}"
+    priv: "{{ app_db_name }}.*:ALL"
+    login_unix_socket: /var/run/mysqld/mysqld.sock
+  no_log: yes
+
+- name: Config database
```
Comment (pluhin) [inline iryna_kurhuzenkava/14.ansible.ws/ans_ws/roles/redmine_db/tasks/main.yaml:14]: Please move this task to app role, and redeploy it to check if everything is correct

**Review by pluhin — CHANGES_REQUESTED** [review]

File: iryna_kurhuzenkava/14.ansible.ws/ans_ws/roles/redmine_app/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    - setup
+
+- name: Configuration files for virtualhost
+  template:
+    src: redmine.site.conf.j2
+    dest: "/etc/apache2/sites-enabled/redmine.site.conf"
+  notify:
+    - apache restart 
+
+- name: Config database
```
Comment (pluhin) [inline iryna_kurhuzenkava/14.ansible.ws/ans_ws/roles/redmine_app/tasks/main.yaml:65]: Why is this here? Please put in correct place. You do deployment of application, and in the end of deployment copy database connection settings. This is wrong

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**